configuration_common.go

```diff
@@ -0,0 +1,29 @@
+package webrtc
+
+import (
+	"strings"
+
+	"github.com/pion/ice"
+)
+
+// getICEServers side-steps the strict parsing mode of the ice package
+// (as defined in https://tools.ietf.org/html/rfc7064) by stripping any
+// erroneous queries from "stun(s):" URLs before parsing.
+func (c Configuration) getICEServers() (*[]*ice.URL, error) {
+	var iceServers []*ice.URL
+	for _, server := range c.ICEServers {
+		for _, rawURL := range server.URLs {
+			if strings.HasPrefix(rawURL, "stun") {
+				// strip the query from "stun(s):" if present
+				parts := strings.Split(rawURL, "?")
```

Comment:
Why not move these changes into `ice.ParseURL()`. Seems like the extra parsing should be inside the parsing function.

Reply:
So me and @mchlrhw were looking at [this rfc](https://tools.ietf.org/html/rfc7064) and these URLs are malformed. However since so many big TURN/STUN providers use them wanted to get rid of this sharp edge.

Maybe the wrong decision, but I pushed to have it in `pion/webrtc` so that we don't introduce the wrongness(?) into `pion/ice`. My thinking was `pion/ice` will be users with a higher level of technical knowledge.

Reply:
Makes sense. Is is malformed just because of the query param? Like these should only apply to stun servers right? https://tools.ietf.org/html/rfc7065#appendix-A

Reply:
Yea! Nice call out, I completely didn't think to check that case.

I have to say I really appreciate the review @trivigy :) it really feels great that we are starting to have a full community around this. Thanks for pushing to keep the quality up, really makes my day to see stuff like this!